editor/animation/animation_blend_tree_editor_plugin.cpp

```diff
@@ -1104,13 +1142,14 @@ void AnimationNodeBlendTreeEditor::_node_renamed(const String &p_text, Ref<Anima
 		return;
 	}
 
-	String prev_name = blend_tree->get_node_name(p_node);
+	String prev_name = p_name;
 	ERR_FAIL_COND(prev_name.is_empty());
 	GraphNode *gn = Object::cast_to<GraphNode>(graph->get_node(prev_name));
 	ERR_FAIL_NULL(gn);
 
 	const String &new_name = p_text;
 
+	ERR_FAIL_COND_MSG(String(p_name).validate_node_name() != p_name, TTR("Invalid AnimationNode name, ignoring rename."));
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(String(p_name).validate_node_name() != p_name, "Invalid AnimationNode name, ignoring rename.");
```
Error messages are not translated